From 39f89277054a1fde00583299a7fc854f0cba24d1 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 29 Jun 2019 16:57:15 +0800
Subject: [PATCH] tweak MAX_TREEINFO_DOWNLOAD_RETRIES

Do not need to retry so many times.

Tweak log debug to log info.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/payload/__init__.py              | 4 ++--
 pyanaconda/payload/install_tree_metadata.py | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/pyanaconda/payload/__init__.py b/pyanaconda/payload/__init__.py
index 8b48625..cbe2154 100644
--- a/pyanaconda/payload/__init__.py
+++ b/pyanaconda/payload/__init__.py
@@ -428,9 +428,9 @@ class Payload(metaclass=ABCMeta):
 
         if self._install_tree_metadata:
             version = self._install_tree_metadata.get_release_version()
-            log.debug("using treeinfo release version of %s", version)
+            log.info("using treeinfo release version of %s", version)
         else:
-            log.debug("using default release version of %s", version)
+            log.info("using default release version of %s", version)
 
         return version
 
diff --git a/pyanaconda/payload/install_tree_metadata.py b/pyanaconda/payload/install_tree_metadata.py
index 30abfd6..04799ce 100644
--- a/pyanaconda/payload/install_tree_metadata.py
+++ b/pyanaconda/payload/install_tree_metadata.py
@@ -27,7 +27,7 @@ from pyanaconda.core import util, constants
 from pyanaconda.anaconda_loggers import get_packaging_logger
 log = get_packaging_logger()
 
-MAX_TREEINFO_DOWNLOAD_RETRIES = 6
+MAX_TREEINFO_DOWNLOAD_RETRIES = 1
 
 
 class InstallTreeMetadata(object):
-- 
2.7.4

